StripeFinancialConnections/StripeFinancialConnections/Source/Native/NativeFlowController.swift

```diff
@@ -314,10 +314,8 @@ extension NativeFlowController {
         }
 
         if showConfirmationAlert {
-            CloseConfirmationAlertHandler.present(
-                businessName: dataManager.manifest.businessName,
-                showNetworkingLanguageInConfirmationAlert: showNetworkingLanguageInConfirmationAlert,
-                didSelectOK: {
+            CloseConfirmationSheetViewController.present(
+                didSelectClose: {
```

Comment:
this code shows how we were able to preserve a very simple interface in terms of being able to present a sheet from the core code